gitlab/base.py

```diff
@@ -137,6 +140,11 @@ def __getattr__(self, name: str) -> Any:
                 )
                 + f"\n\n{_URL_ATTRIBUTE_ERROR}"
             )
+        elif self._lazy:
+            message = f"{message}\n\n" + textwrap.fill(
+                f"{self.__class__!r} was created as a `lazy` object and "
+                f"was not initialzed with any data."
```

Comment:
A typo here and I thought we could maybe mention this is only one of the options here, they could still just have a typo for example:

```suggestion
                f"If you tried to access object attributes returned from the server, "
                f"note that {self.__class__!r} was created as a lazy object and was "
                f"not initialized with any data."
```

Reply:
Done